charts/kof-mothership/templates/sveltos/kof-cluster-role-regional.yaml

```
@@ -0,0 +1,68 @@
+apiVersion: k0rdent.mirantis.com/v1alpha1
+kind: MultiClusterService
+metadata:
+  name: kof-regional-cluster
+spec:
+  clusterSelector:
+    matchLabels:
+      k0rdent.mirantis.com/kof-cluster-role: regional
```

kof-cluster-role label isn't enough as it could be multiple regional clusters
```suggestion
      k0rdent.mirantis.com/kof-cluster-role: regional
      k0rdent.mirantis.com/kof-cluster-class: aws-large
```

---

Probably it's too early to create a multiclusterservice, let's start from a simple service.